src/m365/spo/commands/term/term-list.spec.ts

```diff
@@ -462,7 +462,7 @@ describe(commands.TERM_LIST, () => {
     assert.deepStrictEqual(command.defaultProperties(), ['Id', 'Name', 'ParentTermId']);
   });
 
-  it('gets taxonomy terms including child terms from term set by id, term group by id', async () => {
+  it('gets taxonomy terms including child terms from term set by id, term group by id using includeChildTerms parameter', async () => {
```

Comment:
we should not duplicate tests but have tests only with the new option and just add a new test to check if the deprecation warning message is present